Userland/Libraries/LibCore/File.h

```diff
@@ -109,6 +110,8 @@ class File final : public SeekableStream {
     int m_fd { -1 };
     bool m_last_read_was_eof { false };
     ShouldCloseFileDescriptor m_should_close_file_descriptor { ShouldCloseFileDescriptor::Yes };
+
+    mutable size_t m_file_offset { 0 };
```

Comment:
What in here requires this to be `mutable`? It seems like all functions that modify this variable are not `const`.